lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -112,6 +114,15 @@
                 return this.model;
             },
 
+            getMemberships: function () {
+                if (this.model.has('team')) {
```

Comment:
@peter-fogg told me in person why this change was necessary - since the users are now expanded directly on the membership, we can't just grab them off the team models.